apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/TransformerTestUtils.java

```diff
@@ -970,6 +970,26 @@ static void assertReferenceIdentifierEquals(
         TransformerUtils.retrieveNpiCodeDisplay(expectedIdentifierValue), reference.getDisplay());
   }
 
+  /**
+   * @param expectedIdentifierSystem the expected {@link Identifier#getSystem()} value
+   * @param expectedIdentifierValue the expected {@link Identifier#getValue()} value
+   * @param expectedNpiOrgDisplay the expected {@link Identifier#getValue()} value
+   * @param reference the actual {@link Reference} to verify
+   */
+  static void assertReferenceIdentifierEquals(
+      String expectedIdentifierSystem,
+      String expectedIdentifierValue,
+      Optional<String> expectedNpiOrgDisplay,
+      Reference reference) {
+    assertTrue(reference.hasIdentifier(), "Bad reference: \" + reference");
+    assertEquals(expectedIdentifierSystem, reference.getIdentifier().getSystem());
+    assertEquals(expectedIdentifierValue, reference.getIdentifier().getValue());
+
+    if (expectedNpiOrgDisplay.isPresent()) {
+      assertEquals(expectedNpiOrgDisplay.get(), reference.getDisplay());
+    }
+  }
```

Comment:
Why are we checking in the v1 transformer that we have npiOrg? (i guess we could have a test checking that it _doesnt_ get mapped, but this isnt doing that I think)

Reply:
done